debugging/book-library/script.js

```diff
@@ -1,103 +1,103 @@
+        author.value,
+        pages.value,
+        check.checked
+      );
+      myLibrary.push(book);
+      render();
+    }
   }
-}
 
-function Book(title, author, pages, check) {
-  this.title = title;
-  this.author = author;
-  this.pages = pages;
-  this.check = check;
-}
+  submitButton.addEventListener("click", submit);
 
-function render() {
-  let table = document.getElementById("display");
-  let rowsNumber = table.rows.length;
-  //delete old table
-  for (let n = rowsNumber - 1; n > 0; n-- {
-    table.deleteRow(n);
+  function Book(title, author, pages, check) {
+    this.title = title;
+    this.author = author;
+    this.pages = pages;
+    this.check = check;
   }
-  //insert updated row and cells
-  let length = myLibrary.length;
-  for (let i = 0; i < length; i++) {
-    let row = table.insertRow(1);
-    let cell1 = row.insertCell(0);
-    let cell2 = row.insertCell(1);
-    let cell3 = row.insertCell(2);
-    let cell4 = row.insertCell(3);
-    let cell5 = row.insertCell(4);
-    cell1.innerHTML = myLibrary[i].title;
-    cell2.innerHTML = myLibrary[i].author;
-    cell3.innerHTML = myLibrary[i].pages;
 
-    //add and wait for action for read/unread button
-    let changeBut = document.createElement("button");
-    changeBut.id = i;
-    changeBut.className = "btn btn-success";
-    cell4.appendChild(changeBut);
-    let readStatus = "";
-    if (myLibrary[i].check == false) {
-      readStatus = "Yes";
-    } else {
-      readStatus = "No";
+  function render() {
+    let table = document.getElementById("display");
+    // Clear the previous rows, excluding the header
+    const rowsNumber = table.rows.length;
+    for (let n = rowsNumber - 1; n > 0; n--) {
+      table.deleteRow(n);
     }
-    changeBut.innerHTML = readStatus;
 
-    changeBut.addEventListener("click", function () {
-      myLibrary[i].check = !myLibrary[i].check;
-      render();
-    });
+    // Insert updated rows and cells
+    myLibrary.forEach((book, i) => {
+      let row = table.insertRow();
+      let cell1 = row.insertCell(0);
+      let cell2 = row.insertCell(1);
+      let cell3 = row.insertCell(2);
+      let cell4 = row.insertCell(3);
+      let cell5 = row.insertCell(4);
 
-    //add delete button to every row and render again
-    let delButton = document.createElement("button");
-    delBut.id = i + 5;
-    cell5.appendChild(delBut);
-    delBut.className = "btn btn-warning";
-    delBut.innerHTML = "Delete";
-    delBut.addEventListener("clicks", function () {
-      alert(`You've deleted title: ${myLibrary[i].title}`);
-      myLibrary.splice(i, 1);
-      render();
+      cell1.innerHTML = book.title;
+      cell2.innerHTML = book.author;
+      cell3.innerHTML = book.pages;
+
+      // Add a button for changing read/unread status
+      let changeBut = document.createElement("button");
+      changeBut.className = "btn btn-success";
+      cell4.appendChild(changeBut);
+      let readStatus = book.check ? "No" : "Yes"; 
```

Comment:
**Todo**
Does `book.check == true` mean "Read" or "Not yet read"?

Reply:
yes, it does